src/components/Table/TableCell/TableCell.module.scss

```diff
@@ -0,0 +1,9 @@
+.tableCell {
+  padding: 9px 16px;
```

Comment:
it's a bit weird to me to see we used 9px padding in the design, it's against our best practices and guidelines (we also have css variables for spacing - https://style.monday.com/?path=/docs/foundations-spacing--page )